### PR TITLE
macOS clarity and term updates

### PR DESCRIPTION
**Pull request by timcappalli**
- Removes "iCloud Keychain" and just leaves Apple ID (too wordy)
- Replaces "MacBooks" with "Macs" to cover Mac Mini and iMacs
- Adds a fanout under macOS that highlights that passkeys created on a Mac that are synced to an iOS device can then be used on other platforms via CDA
- Updated blurb at top to reference current year

### CODE REVIEW

File: content/en/device-support/_index.md [hunk 1]
```diff
@@ -10,32 +10,38 @@ weight: 100
...
-- MacBooks using the same Apple ID and iCloud Keychain (synced automatically)
-- iPhones and iPads using the same Apple ID and iCloud Keychain (synced automatically)
+- Macs using the same Apple ID (synced automatically)
+- iPhones and iPads using the same Apple ID (synced automatically)
+  - ...which can then be used on:
+    - Windows devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Chromebooks and other Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Ubuntu devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Macs signed in with a different Apple ID, using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - iPhones and iPads signed in with a different Apple ID, using [FIDO Cross-Device Authentication](/docs/reference/terms/)
```
Comment (MasterKale) [inline content/en/device-support/_index.md:44]: The way these bullet points are nested here gives the impression that only passkeys created in macOS can be used for cross-device authentication...I think these "possibilities of CDA" need their own section because this all applies to passkeys accessible on Android devices too.

File: content/en/device-support/_index.md [hunk 2]
```diff
@@ -10,32 +10,38 @@ weight: 100
...
-Support for passkeys is currently rolling out across major operating systems and browsers, and will continue into 2023. This page will be updated as the ecosystem evolves. The [matrix below](#matrix) maps out the various features that support the passkey experience. Additional information about each platform is available in the [Reference section of Docs](/docs/reference/android).
+Support for passkeys is currently rolling out across major operating systems and browsers. This page will be updated as the ecosystem evolves. The [matrix below](#matrix) maps out the various features that support the passkey experience. Additional information about each platform is available in the [Reference section of Docs](/docs/reference/android).
 
 Passkeys created in **iOS or iPadOS** can be used on:
 
 - The same iPhone or iPad
-- iPhones and iPads using the same Apple ID and iCloud Keychain (synced automatically)
-- MacBooks using the same Apple ID and iCloud Keychain (synced automatically)
-- MacBooks using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- iPhones and iPads using the same Apple ID (synced automatically)
```
Comment (nicksteele) [inline content/en/device-support/_index.md:18]: Would shorten this to 'iOS devices' maybe?

File: content/en/device-support/_index.md [hunk 3]
```diff
@@ -10,32 +10,38 @@ weight: 100
...
 Passkeys created in **Android** can be used on:
 
 - The same Android device
 - Android devices using the same Google account (synced automatically)
-- MacBooks using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- Macs using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - Windows devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
 - iPhones and iPads using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
-- Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- Chromebooks and other Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
```
Comment (nicksteele) [inline content/en/device-support/_index.md:32]: ChromeOS* not Chrome OS

File: content/en/device-support/_index.md [hunk 4]
```diff
@@ -10,32 +10,38 @@ weight: 100
...
 
 Passkeys created in **macOS** can be used on:
 
-- MacBooks using the same Apple ID and iCloud Keychain (synced automatically)
-- iPhones and iPads using the same Apple ID and iCloud Keychain (synced automatically)
+- Macs using the same Apple ID (synced automatically)
+- iPhones and iPads using the same Apple ID (synced automatically)
+  - ...which can then be used on:
+    - Windows devices in Edge and Chrome using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+    - Chromebooks and other Chrome OS devices using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
```
Comment (nicksteele) [inline content/en/device-support/_index.md:41]: Same as above, ChromeOS*

**Review by nicksteele — COMMENTED** [review]
Left editorial comments

File: content/en/device-support/_index.md (hunk 1 above)
Comment (timcappalli) [reply content/en/device-support/_index.md:44]: The intent is to show that passkeys created on macOS, that are accessible on an iOS device signed into the same account, can then be used on those other devices via CDA.

Would tweaking the middle tier bullet help? I don't understand the comment about Android.

File: content/en/device-support/_index.md (hunk 2 above)
Comment (timcappalli) [reply content/en/device-support/_index.md:18]: technically an iPad is not an iOS device, it's an iPadOS device.

File: content/en/device-support/_index.md (hunk 1 above)
Comment (timcappalli) [reply content/en/device-support/_index.md]: see [c1c065f](https://github.com/passkeydeveloper/passkeys.dev/pull/184/commits/c1c065f24e34e71f3d1db820e4873090041843b1)

Comment (MasterKale) [reply content/en/device-support/_index.md]: Looks good to me!

**Review by MasterKale — APPROVED** [review]

**Review by lwe — APPROVED** [review]
Looking good.

The only thing I was contemplating was using _macOS_ instead of _Macs_, but I think the current structure with _operating system_ in titles and using the devices in the text is already consistent, so no need to switch it up right now. 👍
